indexer/prepare.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#===-- prepare.sh ---------------------------------------------------------===//
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+#===----------------------------------------------------------------------===//
+#
+# Fetch Chromium sources and prepare the environment.
+#
+#===----------------------------------------------------------------------===//
+
+set -eux
+
+cd /
+
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

Comment:
Let's add `--depth=1` to speed up the startup time, as we don't care about the history.